web/pgadmin/misc/cloud/utils/__init__.py

```diff
@@ -48,9 +48,8 @@ def _create_server(data):
         name=data.get('name'),
         maintenance_db=data.get('db'),
         username=data.get('username'),
-        ssl_mode='prefer',
         cloud_status=data.get('cloud_status'),
-        connect_timeout=30,
+        connection_params={'sslmode': 'prefer', 'connect_timeout': 30}
```

Comment:
I think `{'sslmode': 'prefer', 'connect_timeout': 30}` can be made common const.